src/common/constants/itemStatuses.js

```diff
@@ -0,0 +1,11 @@
+export const STATUS_IN_PROCESS = 'inProcess';
+export const STATUS_ON_ORDER = 'onOrder';
+export const STATUS_RECEIVED = 'received';
+export const ITEM_STATUS = {
+  [STATUS_IN_PROCESS]: 'In process',
+  [STATUS_ON_ORDER]: 'On order',
+  [STATUS_RECEIVED]: 'Received',
+};
+
+export const CHECK_IN_ITEM_STATUSES = [STATUS_IN_PROCESS];
+export const RECEIVE_ITEM_STATUSES = Object.keys(ITEM_STATUS);
```

Comment:
Status options are the same for receive and check-in, it's the same field, that's why I've asked to leverage it to a separate component

Reply:
but what approach could I use for limiting list for check-in till 1?

Reply:
I mean `CHECK_IN_ITEM_STATUSES` is the same as `RECEIVE_ITEM_STATUSES` from business perspective - it's the same field. So no need to create several vars

Reply:
yes, but for check-in flow we need to provide only one item list...

Reply:
It's not related to Check-in flow. It's related to Item Status field, so it should have the same option list from 1 option

Reply:
I need to ask one more time, does it mean that now we need to provide item status list contains 1 item for both part check-in and receiving?